source/braille.py

```diff
@@ -2468,6 +2468,16 @@ def handleCaretMove(
 		if shouldAutoTether:
 			self.setTether(TetherTo.FOCUS.value, auto=True)
 		if self._tether != TetherTo.FOCUS.value:
+			# However, braille display content is updated in case where:
+			# braille is tethered to review, review cursor does not follow system caret,
+			# and navigator object is focus object or its ancestor.
+			if (
+				not config.conf["reviewCursor"]["followCaret"]
+				and config.conf["braille"]["tetherTo"] == TetherTo.REVIEW.value

```

Comment:
This one is not necessary.
The if check above will already ensure that tethering is set to review, i.e. it is not focus and when tethering is set to automatically, this block won't apply as tethering switched to focus above.
```suggestion
```